src/split-button.test.visuals.ts

```diff
@@ -39,121 +39,129 @@ for (const story of stories) {
           );
         });
 
-        if (story.id.includes('primary-link')) {
```

Comment:
Replaced the conditional with a `test.skip(story.id !== 'split-button--primary-link')` in each test.